docs/modules/theme/pages/fallback-font.adoc

```diff
@@ -95,3 +95,28 @@ font:
 ----
 
 Now Asciidoctor PDF will look for the emoji in the Symbola font instead of the Noto Emoji font.
+
+== Default Microsoft Windows Font Limitations
+When using the fonts provided by default in Microsoft Windows 10, and possibly other versions, it is important to define a fallback font to provide full coverage of Asciidoctor language features.
```

Comment:
Please include an empty line after any section title you add.

Reply:
Implemented in commit 48c098682b30b0afd6feafaa52614244d4f76c0f.